src/web/mjs/connectors/NtsVoidScans.mjs

```diff
@@ -1,12 +1,24 @@
 import WordPressMadara from './templates/WordPressMadara.mjs';
-//dead?
 export default class NtsVoidScans extends WordPressMadara {
 
     constructor() {
         super();
         super.id = 'ntsvoidscans';
-        super.label = 'Void Scans';
+        super.label = 'Void Scans ';
```

Comment:
Why was a space added?
This may lead to problems when sub-folders are created in download directory since windows does not allow a name to end with`.` or ` ` .